### PR TITLE
Fix UnicodeEncodeError failures

### PR DESCRIPTION
**Pull request by lmvasque**
Add _encoding='utf-8'_ flag to file streams & strings to avoid "UnicodeEncodeError: 'ascii' codec can't encode character.." errors.

### CODE REVIEW

**Comment by louismartin** [comment]
Thanks for the PR (related to issue https://github.com/facebookresearch/muss/issues/13).
Do you have a minimal file for repro of the error as input to simplify.py?

**Comment by lmvasque** [comment]
> Thanks for the PR (related to issue #13).
> Do you have a minimal file for repro of the error as input to simplify.py?

Sure, no problem. I do have a failure with the example file in Spanish that is provided with the muss model:

```
python scripts/simplify.py scripts/examples.es --model-name muss_es_mined
File "scripts/simplify.py", line 29, in <module>
print(f'Simplified: {s}')
UnicodeEncodeError: 'ascii' codec can't encode character '\xed' in position 44: ordinal not in range(128)
```

But it can take more time to find the specific lines that failed for the other files that we fixed. I'll let you know if I find another test case.

File: scripts/simplify.py [hunk 1]
```diff
@@ -24,6 +24,8 @@
     source_sentences = read_lines(args.filepath)
     pred_sentences = simplify_sentences(source_sentences, model_name=args.model_name)
     for c, s in zip(source_sentences, pred_sentences):
+        s = s.encode('utf-8')
+        c = c.encode('utf-8')
```
Comment (sadakmed) [inline scripts/simplify.py:28]: There's no need for encoding here as they will be already encoded with `read_lines` function

File: setup.py [hunk 2]
```diff
@@ -8,10 +8,10 @@
 from setuptools import setup, find_packages
 
 
-with open('README.md', 'r') as f:
+with open('README.md', 'r', encoding='utf-8') as f:
     long_description = f.read()
 
-with open('requirements.txt', 'r') as f:
+with open('requirements.txt', 'r', encoding='utf-8') as f:
```
Comment (sadakmed) [inline setup.py:14]: no need here, library names are safe from such issues

**Review by sadakmed — CHANGES_REQUESTED** [review]

**Pull request merged**